tensorflow/python/training/checkpoint_management.py

```diff
@@ -347,15 +347,11 @@ def latest_checkpoint(checkpoint_dir, latest_filename=None):
   return None
 
 
-@deprecation.deprecated(
-    date=None,
-    instructions="Use standard file APIs to check for files with this prefix.")
-@tf_export(v1=["train.checkpoint_exists"])
-def checkpoint_exists(checkpoint_prefix):
+def cp_exists(checkpoint_prefix):
```

Comment:
How about checkpoint_exists_internal; cp is pretty ambiguous.

Otherwise looks great.

Reply:
Just renamed the function.